blazor/rating/appearance.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Appearance in Blazor Rating Component | Syncfusion
+description: Checkout and learn here all about Appearance with Syncfusion Blazor Rating component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Rating
+documentation: ug
+---
+
+# Appearance in Blazor Rating Component
+
+You can also customize the appearance of rating control.
+
+## Items count
+
+You can specify the number of rating items using the [ItemsCount](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_ItemsCount) property.
+
+Below example demonstrates the Items Count of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating ItemsCount="8"></SfRating>
+
+```
+
+![Blazor Rating Component with Items Count](images/blazor-rating-itemscount.png)
+
+## Disabled
+
+You can disabled the rating component by using the [Disabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Disabled) property. The user will not be able to interact with it.
+
+Below example demonstrates the Disabled of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating Disabled=true></SfRating>
+
+```
+
+![Blazor Rating Component with Disabled](images/blazor-rating-disabled.png)
+
+## Visible
+
+By using the [Visible](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Visible) property. You can customize the visibility of the Rating. If the value of Visible is true, then the rating component is in a visible state.
+
+Below example demonstrates the Visible of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating Visible=true></SfRating>
+
+```
+
+![Blazor Rating Component with Visible](images/blazor-rating-full-precision.png)
+
+## Read Only
+
+By using the [ReadOnly](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_ReadOnly) property, The user will not be able to interact with the rating item and change the rating value. 
+
+Below example demonstrates the Read Only of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating ReadOnly=true></SfRating>
+
+```
+
+![Blazor Rating Component with Visible](images/blazor-rating-full-precision.png)
+
+## Css Class
+
+You can customize the appearance of the rating component, such as by changing its colors, fonts, sizes, or other visual aspects by using the [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_CssClass) property. 
+
+Below example demonstrates Css Class of Rating.
+
+```cshtml
+
+<SfRating>
```

Comment:
Why are you adding a template example in the CSS class topic??
Add an appropriate example for using the "CSSClass" property like, changing the border color, changing the space between stars, changing the size of star icon, adding background color for the rating, and so on

Reply:
I have updated it as per comments.